Site/bc_weather_station_dashboard/website/static/CSS/precipitation_card.css

```diff
@@ -0,0 +1,82 @@
+#precipitation-card {
+    width: 180px;
+}
+
+.precipitation-container {
+    position: relative;
+    width: 115px;
+    height: 160px;
+    display: flex;
+    justify-content: center;
+    border-bottom: 3px solid rgba(255, 255, 255, 0.1);
+    margin-bottom: 10px;
+}
+
+.precipitation-container .cloud {
+    position: relative;
+    width: 150px;
+    height: 40px;
+    top: 30px;
+    background: #fff;
+    border-radius: 100px;
+    filter: drop-shadow(8px 8px 0 #0005);
+    animation: animateCloud 3s linear infinite;
+}
+
+@keyframes animateCloud {
```

Comment:
Maybe only flash the cloud when it is really down pouring?

Reply:
Good idea, I'll add that into the issues for this week!